design/user_types.go

```diff
@@ -104,3 +104,33 @@ var UpdateTrackerQueryAlternatePayload = a.Type("UpdateTrackerQueryAlternatePayl
 
 	a.Required("query", "schedule", "trackerID")
 })
+
+// Users represents a user object (TODO: add better description)
+var Users = a.Type("Users", func() {
```

Comment:
The same as above. Why duplicating Users and UserAttrbiutes? We really should re-use the same types we use in other places.